installer/docker/Dockerfile

```diff
@@ -1,8 +1,12 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
```

Comment:
isn't this a pretty old distribution at this point? Maybe 22.04? It has LTS ... I know it doesn't matter if it's in docker but potentially easier for future developers to update this file if it's not too outdated?

Reply:
Yeah that's a good point, at first I was trying to change as little as possible I worried the newest ubuntu image might result in other dependencies having to be updated, thanks for pointing it out!

Reply:
So I do run into an error with ubuntu:22.04, seems like the package `libreadline5` is not available, given the timeframe of the demonstration next week, i'm going to focus on testing the simulation with what I got so far (so using ubuntu20.04). if everything is fine, we can use it for the demo, and make any further upgrades afterwards, thanks for point it out though @jasmainak

Reply:
We can discuss in our zoom meeting if it makes sense to update to 22.04, I'm pretty impartial since we're aiming to use this primarily for workshops (and this version of the software isn't being further updated)